src/plugins/cosmos/components/modals/Staking/views/Overview.tsx

```diff
@@ -100,50 +99,52 @@ export const Overview: React.FC<StakedProps> = ({
               apr={bnOrZero(validatorInfo?.apr)}
             />
           </Skeleton>
-          <Skeleton isLoaded={isLoaded} width='100%' mb='40px' justifyContent='space-between'>
-            <Box width='100%'>
-              <Text translation={'defi.rewards'} mb='12px' color='gray.500' />
-              <AssetClaimCard
-                assetSymbol={asset.symbol}
-                assetIcon={asset.icon}
-                cryptoRewardsAmount={bnOrZero(rewardsAmount)
-                  .div(`1e+${asset.precision}`)
-                  .decimalPlaces(asset.precision)}
-                fiatRate={bnOrZero(marketData.price)}
-                renderButton={() => (
-                  <ClaimButton
-                    assetId={assetId}
-                    validatorAddress={validatorAddress}
-                    // We're getting fractions of uatom as rewards, but at protocol-level, it is actually impossible to claim these
-                    // Any amount that's less than 1 uatom effectively means no rewards
-                    isDisabled={bnOrZero(rewardsAmount).lt(1)}
+        </Flex>
+      </ModalBody>
+      <ModalFooter flexDirection='column'>
+        <Skeleton isLoaded={isLoaded} width='100%' justifyContent='space-between'>
+          <Stack width='100%'>
+            <Text translation={'defi.rewards'} color='gray.500' />
+            <AssetClaimCard
+              assetSymbol={asset.symbol}
+              assetIcon={asset.icon}
+              cryptoRewardsAmount={bnOrZero(rewardsAmount)
+                .div(`1e+${asset.precision}`)
+                .decimalPlaces(asset.precision)}
+              fiatRate={bnOrZero(marketData.price)}
+              renderButton={() => (
+                <ClaimButton
+                  assetId={assetId}
+                  validatorAddress={validatorAddress}
+                  // We're getting fractions of uatom as rewards, but at protocol-level, it is actually impossible to claim these
+                  // Any amount that's less than 1 uatom effectively means no rewards
+                  isDisabled={bnOrZero(rewardsAmount).lt(1)}
+                />
+              )}
+            />
+          </Stack>
+        </Skeleton>
+        {shouldDisplayUndelegationEntries && (
+          <Skeleton isLoaded={isLoaded} width='100%' minHeight='68px'>
```

Comment:
```suggestion
          <Skeleton isLoaded={isLoaded} width='100%' minHeight='68px' mt={4}>
```

![image](https://user-images.githubusercontent.com/14963751/166881067-3c593311-2586-4466-94bb-5b738e6f79f7.png)

There is a small space problem there, as this div is inside a condition, I would probably add the margin on it rather than on the top one

Reply:
Also, we should unify the spacing between `Rewards` and `Unstaking` titles and their content I guess